default/scripting/ship_parts/FighterHangar/FT_HANGAR_1.focs.txt

```diff
@@ -23,7 +23,12 @@ Part
             activation = Source
             stackinggroup = "INTERCEPTOR_FAST_LAUNCH_EFFECT"
             effects = [
-                SetMaxCapacity partname = "FT_BAY_1" value = Value + 2
+                SetMaxCapacity partname = "FT_BAY_1" value = (PartsInShipDesign name = "FT_BAY_1" design = Target.DesignID) * (
```

Comment:
This seems to be buggy. Launch capacity ends up being **3**+(Statistic If...) * PartsInShipDesign **^2**
Reported [here](https://www.freeorion.org/forum/viewtopic.php?f=6&t=11352&p=99460#p99459).